tests/common/helpers/platform_api/chassis.py

```diff
@@ -90,6 +90,8 @@ def get_all_modules(conn):
 def get_module(conn, index):
     return chassis_api(conn, 'get_module', [index])
 
+def get_module_index(conn, mod_name):
+    return chassis_api(conn, 'get_module_index', [mod_name])
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be two blank lines between top-level function definitions.